test/integration_test.py

```diff
@@ -562,7 +562,14 @@ def worker(_):
                             response = self._check_endpoint(PUT, '/manifest/files', args=args, fetch=fetch)
                             self._manifest_validators[format](catalog, response)
 
-                        num_workers = 3
+                        # Number of workers reduced to one as a workaround to a
+                        # race condition which allows the manifest service to
+                        # attempt to start an execution that has already
+                        # completed.
+                        #
+                        # https://github.com/DataBiosphere/azul/issues/6850
+                        #
+                        num_workers = 1
```

Comment:
As per our long-established conventions, this should be a FIXME and you don't need to explain as much since all of that is on the ticket. So

```suggestion
                        # FIXME: Set number of workers back to 3
                        #        https://github.com/DataBiosphere/azul/issues/6850
                        num_workers = 1
```